expression/builtin.go

```diff
@@ -723,3 +723,11 @@ var funcs = map[string]functionClass{
 	ast.TiDBIsDDLOwner: &tidbIsDDLOwnerFunctionClass{baseFunctionClass{ast.TiDBIsDDLOwner, 0, 0}},
 	ast.TiDBParseTso:   &tidbParseTsoFunctionClass{baseFunctionClass{ast.TiDBParseTso, 1, 1}},
 }
+
+// IsBuiltinFuncName check if given function name is a builtin sql function
+func IsBuiltinFuncName(name string) bool {
```

Comment:
I don't think we really need this function, `builtin function` is a concept contract to `user define function`, since TiDB doesn't support UDF, we can directly check  `if _, ok := funcs[name]; ok { ... }`

Reply:
Yeah, but I don't want to export variable `funcs`, and only use `funcs` for this case.

Reply:
How about `IsFunctionSupported()`?